samples/subsys/event_manager/src/events/config_event.c

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+
+#include "config_event.h"
+
+
+static void log_config_event(const struct event_header *eh)
+{
+	struct config_event *event = cast_config_event(eh);
```

Comment:
maybe a macro would be more useful/readable (aligning with `CONTAINER_OF` usage), e.g.

```c
struct config_event *event = EVM_CAST(eh, config);
```

Reply:
good idea - we could follow this up along with new_event api @zycz